### DISCUSSION TITLE
What are your plans for the future Ben?

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#2907] [General]
No, this time I'm not complaining about Google messing up your notebooks. :) I was wondering what are your plans for the future of your repositories, now that things are getting pretty exciting with Flux. Do you know if Automatic1111 will get support for Flux anytime soon? To be honest, what I would love is to find a way to finetune XL and Flux via Google Pro because I would never have a powerful enough computer to do it and I just fell in love with Flux at first sight. It's like we are getting back in time, I love watching people posting their first Flux loras on Reddit. And I feel envious just like I felt years ago watching those cool Disney and Ghibli Loras popping out from nowhere. Do whatever you want, of course, I am just curious and excited. (And if you know any colab to finetune XL tell me for the love of God!)

**Comment by aifantasyglam** [comment]
Hello everyone

@TheLastBen 
Regarding this post, I have a similar doubt, because I'm not sure how many people are in the same situation, but in my case, I don´t have a powerful pc, so I enjoy Colab Pro and Last Ben´s Notebook to use A1111 to generate images...

I´m afraid of the day when this Notebook being abandoned completely😔 because is really cool, even nowaday that I enjoy Flux too.

**Reply by TheLastBen** [reply]
A1111 colab will always be maintained

**Reply by aifantasyglam** [reply]
> A1111 colab will always be maintained

that´s a good news 😭

**Reply by ClaraSanders** [reply]
What about Dreambooth? Will you still support it? Your repo is the only way for me to fully fine-tune models. Yours is probably the only one left after Linaqruf stopped updating Kohya. I've been looking for MONTHS, asking everywhere on Discord and Reddit with zero luck. I would love to find a way to fully fine-tune XL models but for now I can only fine-tune 1.5 thanks to you and I can make XL and Flux Loras thanks to Strawberry's Kohya's notebooks and FluxGym. In fact, if there is a way to implement XL support to your Dreambooth I would pay you to do it, I am this desperate.

**Reply by TheLastBen** [reply]
@ClaraSanders there sdxl trainer on my paperspace template

**Comment by Khayoon** [comment]
@aifantasyglam @ClaraSanders What are you using to run flux on colab let alone fine tune it?

**Reply by aifantasyglam** [reply]
Hi, I don´t try Flux on Colab yet, I use Civitai and TensorArt to work with Flux
